Assets/Shaders/SplatCube.shader

```diff
@@ -0,0 +1,207 @@
+Shader "Custom/CubeShader"
+{
+    Properties
+    {
+        // splat map
+        _SliceCount ("SliceCount", Range(1,256)) = 256
+        _SplatMap ("SplatMap", 2D) = "black" {}
+
+        // detail maps
+        _DetailUvScale ("DetailUvScale", Float) = 5.0
+        _AlbedoMaps ("AlbedoMaps", 2DArray) = "" {}
+        _NormalMaps ("NormalMaps", 2DArray) = "" {}
+        _RoughMaps ("RoughMaps", 2DArray) = "" {}
+        _CavityMaps ("CavityMaps", 2DArray) = "" {}
+
+        // noise map
+        _UvNoiseScale("UvNoiseScale", Float) = 15.0
+        _UvNoiseRadius("UvNoiseRadius", Float) = 5.0
+        _NoiseMap("NoiseMap", 2D) = "black" {}
```

Comment:
Will be removed

Reply:
https://github.com/davidasberg/cubes_traveller/pull/11/commits/08b629e7dd2ecaeffb23cb3b893479de655ce7ed